.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+sudo: false
+python:
+- '2.7'
+- '3.3'
```

Comment:
We make no claim on python3 compatibility. In fact, webapp2 states that it is not python3 compatible, but I have never tried. Also, why 3.3 and not 3.5?